packages/near-fast-auth-signer/src/lib/controller.ts

```diff
@@ -199,6 +226,7 @@ class FastAuthController {
     if (!keypair) {
       keypair = KeyPair.fromRandom('ED25519');
       await this.keyStore.setKey(this.networkId, `oidc_keypair_${oidcToken}`, keypair);
+      await this.localStore.setKey(this.networkId, `oidc_keypair_${oidcToken}`, keypair);
```

Comment:
Just for curiosity, why we need the keyStore and localStore? Isn't bad to have to keep to places in sync and have two sources of truth?

Reply:
Good point. The design of this implementation needs both ephemeral in memory keystore and the local storage keystore that persists between sessions. The in memory keystore will store FAKs after retrieving from webauthn and the local storage keystore will store a permanent mapping of oidc token to claimed keypair so that we can sign things with the recovery service permanently in non webauthn browsers. I believe it's possible for us to move the oidctoken <> keypair mapping permanently to localstorage removing it from in memory storage since we are storing it there now anyway. Will do some further testing and circle back on this